lib/mixins/dropdown.js

```diff
@@ -81,6 +80,9 @@ export default {
                         el.addEventListener('mouseover', this.noop);
                     });
                 }
+
+                document.addEventListener('keydown', this.keydownListener);
```

Comment:
Lines 37 & 38 of the template also add a keydown listener handler for keyboard navigation